README.md

```diff
@@ -1,298 +1,7 @@
-  - If you'd like to use Airflow 1.10 operators, see the [`samples/pipeline.airflow1.yaml`](https://github.com/GoogleCloudPlatform/public-datasets-pipelines/blob/main/samples/pipeline.yaml) as a reference.
-
-
-# Best Practices
-
-- When your tabular data contains percentage values, represent them as floats between 0 to 1.
-- To represent hierarchical data in BigQuery, use either:
-  - (Recommended) Nested columns in BigQuery. For more info, see [the documentation on nested and repeated columns](https://cloud.google.com/bigquery/docs/nested-repeated).
-  - Or, represent each level as a separate column. For example, if you have the following hierarchy: `chapter > section > subsection`, then represent them as
-
-    ```
-    |chapter          |section|subsection          |page|
-    |-----------------|-------|--------------------|----|
-    |Operating Systems|       |                    |50  |
-    |Operating Systems|Linux  |                    |51  |
-    |Operating Systems|Linux  |The Linux Filesystem|51  |
-    |Operating Systems|Linux  |Users & Groups      |58  |
-    |Operating Systems|Linux  |Distributions       |70  |
-    ```
-
-- When running `scripts/generate_terraform.py`, the argument `--bucket-name-prefix` helps prevent GCS bucket name collisions because bucket names must be globally unique. Use hyphens over underscores for the prefix and make it as unique as possible, and specific to your own environment or use case.
-- When naming BigQuery columns, always use `snake_case` and lowercase.
-- When specifying BigQuery schemas, be explicit and always include `name`, `type` and `mode` for every column. For column descriptions, derive it from the data source's definitions when available.
-- When provisioning resources for pipelines, a good rule-of-thumb is one bucket per dataset, where intermediate data used by various pipelines (under that dataset) are stored in distinct paths under the same bucket. For example:
-
-  ```
-  gs://covid19-tracking-project-intermediate
-      /dev
-          /preprocessed_tests_and_outcomes
-          /preprocessed_vaccinations
-      /staging
-          /national_tests_and_outcomes
-          /state_tests_and_outcomes
-          /state_vaccinations
-      /prod
-          /national_tests_and_outcomes
-          /state_tests_and_outcomes
-          /state_vaccinations
-
-  ```
-  The "one bucket per dataset" rule prevents us from creating too many buckets for too many purposes. This also helps in discoverability and organization as we scale to thousands of datasets and pipelines.
-
-  Quick note: If you can conveniently fit the data in memory, the data transforms are close-to-trivial and are computationally cheap, you may skip having to store mid-stream data. Just apply the transformations in one go, and store the final resulting data to their final destinations.
+More details about this repository can be found in the [Wiki Pages](https://github.com/GoogleCloudPlatform/public-datasets-pipelines/wiki).
```

Comment:
Let's say

For detailed documentation, please see the [wiki](https://github.com/GoogleCloudPlatform/public-datasets-pipelines/wiki).`